osrf/vrx
# Changing hector config to use Gazebo world SDF

This came up in PR #341 .

This PR makes use of the  the Hector `GazeboRosGPS`  a parameter `useWorldSphericalCoordinates` so that the same geographic reference is used by Gazebo and by the GPS plugin.

Here is what I did to verify:

Start with USV at gazebo origin:

```
roslaunch vrx_gazebo sydneyregatta.launch x:=0 y:=0 Y:=0
```

* Verify that at gazebo origin, GPS plugin provides lat/lon very near the specified spherical_coordinates lat/lon.

* Manually move WAM-V to roughly Gazebo x = 100, y = 0, verify that the Hector plugin reports that the latitude is roughly the same and that the longitude increases, as expected.

* Repeat for each quadrant.
